packages/adblocker/test/matching.test.ts

```diff
@@ -396,6 +396,23 @@ describe('#NetworkFilter.match', () => {
       url: 'https://sub.y.com/bar',
       type: 'script',
     });
+
+    // to
```

Comment:
Should we add more tests? Not sure if we cover all corner cases with these 3 tests?